src/variables/body/index.tsx

```diff
@@ -60,6 +113,24 @@ const VariableComponent = ({ model }: { model: Variables.Model }) => {
   return <>{List()}</>;
 };
 
+const convertType = (variable: Variables.IVariable) => {
+  const type = variable.type;
+  let value: any = variable.value;
+  if (type === 'int' || type === 'float') {
+    return value * 1;
+  }
+  if (type === 'bool') {
+    return value === 'False' ? false : true;
+  }
+  if (type === 'str') {
+    return (value as string)
+      .split('')
+      .slice(1, value.length - 1)
+      .join('');
+  }
```

Comment:
I was wondering why we had to do that, but that's because strings are returned with single quotes:

![image](https://user-images.githubusercontent.com/591645/68770730-fd838400-0626-11ea-872a-753ff9fb2163.png)

Reply:
We should be able to call `.slice(1, value.length - 1)` directly on the string.

Reply:
yeah you have right.

Reply:
I just pushed a small refactoring :+1: